server/types/Search.ts

```diff
@@ -4,3 +4,7 @@ import { SearchOrderFormData } from '../models/form-data/searchOrder'
 export type SearchFormInput = Token & {
   data: SearchOrderFormData
 }
+
+export type SearchResultsRequest = Token & {
```

Comment:
Should this be:
```suggestion
export type SearchResultsRequest = Token & QueryExecutionResponse
```
instead of
```
{
  queryExecutionId: string
}
```

Reply:
Updated to use the interface QueryExecutionResponse interface.